src/utils/__tests__/remove-nodes-from-original-code.spec.ts

```diff
@@ -23,7 +24,7 @@ test('it should remove nodes from the original code', () => {
     const ast = babelParser(code, { sourceType: 'module' });
     const importNodes = getImportNodes(code);
     const sortedNodes = getSortedNodes(importNodes, {
-        importOrder: [],
+        importOrder: testingOnly.normalizeImportOrderOption([]),
```

Comment:
Instead of exporting this for testing, we could consider "pre-normalizing" all of the arrays used here in these tests (i.e. making sure they all specify builtins and third party).  Don't feel strongly about it, only that exporting code only to use in testing feels like a smell.

Reply:
- pre-normalizing might let subsequent developers think those fields are optional.
- I guess I wanted to preserve the developer intent: "I intend to provide an empty array to the plugin" -- and the normalization is an implementation detail of that?